src/peer_transport/handler.ts

```diff
@@ -133,5 +135,21 @@ export async function initiateConnection ({ rtcConfiguration, signal, stream: ra
 
   await readCandidatesUntilConnected(connectedPromise, pc, stream)
   channel.close()
-  return [pc, muxerFactory]
+
+  const remoteAddress = parseRemoteAddress(pc.currentRemoteDescription?.sdp ?? '')
+
+  return { pc, muxerFactory, remoteAddress }
+}
+
+function parseRemoteAddress (sdp: string): string {
+  // 'a=candidate:1746876089 1 udp 2113937151 0614fbad-b...ocal 54882 typ host generation 0 network-cost 999'
+  const candidateLine = sdp.split('\r\n').filter(line => line.startsWith('a=candidate')).pop()
+  const candidateParts = candidateLine?.split(' ')
+
+  if (candidateLine == null || candidateParts == null || candidateParts.length < 5) {
+    log('could not parse remote address from', candidateLine)
+    return '/webrtc'
+  }
+
+  return `/dnsaddr/${candidateParts[4]}/${candidateParts[2]}/${candidateParts[3]}/webrtc`
```

Comment:
Why are we returning the `remoteAddress` now? Would this be a breaking change?

Reply:
This is fix.  Previously we returned the relayed address, so it looked like the remote webrtc connection had an address like `/ip4/123.123.132.123/p2p/QmRelay/webrtc/p2p/QmPeer` when in fact only the SDP handshake occurred over that address, the actual connection was over webrtc.

This can be validated by connecting two browser peers via a relay then stopping the relay - the connection is not tunnelled via the relay so it shouldn't be in the remote addr of the connection.